custom/icds_reports/static/js/directives/additional-filter/additional-filter.directive.js

```diff
@@ -152,19 +154,14 @@ function AdditionalFilterController($scope, $location, $uibModal, storageService
 AdditionalFilterController.$inject = ['$scope', '$location', '$uibModal', 'storageService', 'genders', 'ages'];
 AdditionalModalController.$inject = ['$location', '$uibModalInstance', 'filters', 'genders', 'ages', 'agesServiceDeliveryDashboard', 'haveAccessToFeatures'];
 
-window.angular.module('icdsApp').directive("additionalFilter", ['templateProviderService', function (templateProviderService) {
-    var url = hqImport('hqwebapp/js/initial_page_data').reverse;
-    return {
```

Comment:
some variables are removed from the return. is that related to upgrade or a cleanup?

Reply:
its part of upgrade. Components doesn't need `restrict`, `bindToController`